docs/sources/docker-hub-enterprise/configuration.md

```diff
@@ -258,7 +258,7 @@ API.
 * *Yaml configuration file*: This file (`/usr/local/etc/dhe/storage.yml`) is
 used to configure the image storage services. The editable text of the file is
 displayed in the dialog box. The schema of this file is identical to that used
-by the [Registry 2.0](http://docs.docker.com/registry/configuration/).
+by the [Registry 2.0](https://docs.docker.com/registry/configuration/).
```

Comment:
This file is copied from one of our private repos which you don't have access to.  The change actually has to be made there or else the next copy over will overwrite your change. ping @SvenDowideit @fredlf for the fix on dhe repo.